src/test/java/uk/gov/hmcts/ccd/domain/service/stdapi/CallbackInvokerTest.java

```diff
@@ -36,10 +39,9 @@
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.*;
 import static org.junit.jupiter.api.Assertions.assertNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.Mockito.*;
 
 class CallbackInvokerTest {
```

Comment:
I think we are making too much use of integration tests. According to the testing pyramid principle, most tests should be unit tests. Then fewer higher level tests like integration tests. If you do that, we should be able to minimise the number of json files we adding to the stub service to just few main examples